packages/core/src/utils.js

```diff
@@ -1265,3 +1265,115 @@ export function schemaRequiresTrueValue(schema) {
 
   return false;
 }
+
+export function getRegistry(props) {
```

Comment:
We should move all of these new functions in utils.js to just Form.js (because they're  quite Form-specific)